plugins/webp-uploads/hooks.php

```diff
@@ -747,13 +747,84 @@ function webp_uploads_render_generator(): void {
 }
 add_action( 'wp_head', 'webp_uploads_render_generator' );
 
+/**
+ * Process a block's content to handle background images for specific block types.
+ *
+ * This function targets blocks like Cover and Group that may use background images,
+ * converting them to modern image formats when appropriate.
+ *
+ * @since n.e.x.t
+ *
+ * @param string               $block_content The block content.
+ * @param array<string, mixed> $block         The block.
+ * @return string The filtered block content.
+ */
+function webp_uploads_filter_block_background_images( string $block_content, array $block ): string {
```

Comment:
Since plugins can filter this value to be anything, it's not safe to add a PHP type hint since a fatal error could happen. So it's better to allow for `mixed`:

```suggestion
 * @param string|mixed         $block_content The block content.
 * @param array<string, mixed> $block         The block.
 * @return string The filtered block content.
 */
function webp_uploads_filter_block_background_images( $block_content, array $block ): string {
	// Because plugins can do bad things.
	if ( ! is_string( $block_content ) ) {
		$block_content = '';
	}

```